__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require("supertest");
+const app = require("../app")
+const seed = require("../db/seeds/seed")
+const data = require('../db/data/test-data')
+const db = require('../db/connection')
+const endpoints = require('../endpoints.json')
+const topics = require('../db/data/test-data/topics')
+
+beforeEach(() => seed(data))
+afterAll(() => db.end())
+
+describe('GET /api/topics', () => {
+    it('200 - should respond with an array of topics', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            expect(body).toEqual(topics)
+        })
+    })
+    it('200 - each topic should have a slug and description key', () => {
+        return request(app)
+        .get('/api/topics')
+        .expect(200)
+        .then(({body}) => {
+            const allTopics = body
+            allTopics.forEach((topic) => {
```

Comment:
if you are testing multiple things using a forEach please test the length of the array before you use the forEach. this way if you have an empty array the tests inside your forEach would be skipped but the test for the length would still work.